ci-operator/step-registry/baremetalds/packet/setup/baremetalds-packet-setup-ref.yaml

```diff
@@ -8,5 +8,19 @@ ref:
       memory: 600Mi
     limits:
       memory: 2Gi
+  env:
+  - name: PACKET_ACCOUNT
```

Comment:
a little bit of name bikeshedding: I found it a little bit misleading, since it's not properly a Packet account, but an (indirect) reference to an `Equinix` `project`, so I'd suggest to use something like `PACKET_PROJECT` or `EQUINIX_PROJECT` (in case in future the other CI references will ever be updated...) or keep it neutral with `METAL_PROJECT`

Reply:
Or eventually `CLUSTER_OWNER` (see https://github.com/openshift/ci-tools/pull/2268)